src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -784,7 +784,11 @@ UINT_PTR ExceptionTracker::FinishSecondPass(
     {
         CopyOSContext(pThread->m_OSContext, pContextRecord);
         SetIP(pThread->m_OSContext, (PCODE)uResumePC);
+#ifdef TARGET_UNIX
+        uAbortAddr = NULL;
```

Comment:
Nit: `uAbortAddr` is already initialized to the zero value, so this assignment is not needed.